main/models/models.py

```diff
@@ -366,6 +366,12 @@ class Meta:
         null=True,
         blank=True,
     )
+    ignored_duplicates = models.ManyToManyField(
+        "Resource", blank=True, related_name="ignored_duplicates_by_ressources"
```

Comment:
```suggestion
        "self", blank=True, related_name="ignored_duplicates_by_ressources"
```
me semble un poil plus logique, mais ça change rien